README.md

```diff
@@ -20,119 +20,114 @@ This extension requires the following NPM packages to be installed either locall
+**Notes**:
 
-3. Open a JavaScript (`.js` or `.jsx`) or TypeScript (`.ts` or .`tsx`) file
-4. Press `CTRL + SHIFT + P` (Windows/Linux) or `CMD + SHIFT + P` (macOS) to open the command palette
-5. Start typing `Format Document With` and select `Prettier ESLint`
-6. Click on `Output` to open the panel
-7. If you see _Error: Cannot find module_, quit and restart Visual Studio Code
-8. If restarting did not work:
-   - If you saw _Error: Cannot find module '@typescript-eslint/parser'_, view the [TypeScript project](https://github.com/idahogurl/vs-code-prettier-eslint/tree/master/examples/typescript) for a working example.
-   - Make sure you have the required packages installed locally (global installations don't work sometimes)
+- The [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) is **not** required.
 
-**Note**: Most issues are caused by using an unsupported ESLint version or an invalid ESLint configuration.
+- The [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) is **not** required. However, it is needed to have lint errors show while editing your file.
 
-Running your files through the [Prettier ESLint CLI](https://github.com/prettier/prettier-eslint-cli) first is a good way to determine if it's the extension, the `prettier-eslint` package, or your configuration. 
+## Install dependencies
 
-You can run `eslint --print-config .eslintrc.js` to check your ESLint configuration. This prints out the configuration being used after it applies plugins & extended configurations.
+First we need to install `prettier` and `eslint` as dev dependencies in your project. Run one of the following commands based on your project requirements. The commands listed below use `yarn` but you can also use `npm`. Just replace `yarn add` with `npm i`
 
-**Note**: There are known performance issues with quad-core or slower processors. The slowness comes from ESLint CLI which the 'prettier-eslint' package because uses.
+```bash 
+## minimum requirements
+yarn add -D prettier@^2.5.1 eslint@^8.7.0
 
-## Installation
+## if your project uses typescript
+yarn add -D prettier@^2.5.1 eslint@^8.7.0 @typescript-eslint/parser@^5.0.1 typescript@^4.4.4
+```
 
-**Notes**:
+## Project settings 
 
-- The [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) is not required.
+Next we need to configure your project to use the extension. To do that, we're going to open or create a `settings.json` file at the root of your project. If you already have a `.vscode/settings.json` file in your project, you can skip the first two steps below and jump straight to step 3. 
 
-- The [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) is not required. However, it is needed to have lint errors show while editing your file.
+1. Open the command palette in vscode by typing:
```

Comment:
```suggestion
1. Open the command palette in VS Code by typing:
```

This is how Microsoft abbreviates Visual Studio Code. See screenshot below.
<img width="1435" alt="Screen Shot 2022-07-20 at 10 08 53 AM" src="https://user-images.githubusercontent.com/10620169/180030856-4fcb70de-6d25-480c-8360-bb6df51efcc5.png">